spec/fixtures/checksum_root01/moab_storage_trunk/INVALID_MOABS_HERE.md

```diff
@@ -0,0 +1 @@
+# Invalid Moabs Here.
```

Comment:
maybe add a bit of explanation (e.g. druid paths don't match druids in manifests).

or not, i don't feel strongly about this at all.